StyleCop.Analyzers/StyleCop.Analyzers/Lightup/LightupHelpers.cs

```diff
@@ -134,6 +136,11 @@ internal static bool CanWrapNode(SyntaxNode node, Type underlyingType)
 
             var propertySyntaxType = property.PropertyType.GenericTypeArguments[0];
 
+            if (!ValidatePropertyType(typeof(TProperty), propertySyntaxType))
+            {
+                throw new InvalidOperationException();
```

Comment:
:memo: It doesn't appear this exception case is covered by any tests (looking at codecov)